src/sage/functions/log.py

```diff
@@ -1106,30 +1131,30 @@ def _eval_(self, z, m):
         """
         EXAMPLES::
 
-            sage: harmonic_number(x,0)
+            sage: harmonic_number(x, 0)                                                 # needs sage.symbolic
             x
-            sage: harmonic_number(x,1)
+            sage: harmonic_number(x, 1)                                                 # needs sage.symbolic
             harmonic_number(x)
-            sage: harmonic_number(5)
+            sage: harmonic_number(5)                                                    # needs mpmath
             137/60
-            sage: harmonic_number(3,3)
```

Comment:
Really `mpmath`? 

I am guessing `mpmath` is needed only for numerical computation. No?

Reply:
Right. I've corrected it to `# needs sage.symbolic`